src/vmm/src/snapshot.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please move this file to src/vmm/src/vmm_config.rs.

Reply:
I didn't want to add it there initially because I was thinking it will contain some functionality that won't fit in `vmm_config`. But in its current form, it seems better to move it there. Done.